x-pack/legacy/plugins/maps/public/layers/util/can_skip_fetch.js

```diff
@@ -128,3 +128,24 @@ export async function canSkipSourceUpdate({ source, prevDataRequest, nextMeta })
     && !updateDueToPrecisionChange
     && !updateDueToSourceMetaChange;
 }
+
+export function canSkipStyleMetaUpdate({ prevDataRequest, nextMeta }) {
+  if (!prevDataRequest) {
+    return false;
+  }
+  const prevMeta = prevDataRequest.getMeta();
+  if (!prevMeta) {
+    return false;
+  }
+
+  const updateDueToFields = !_.isEqual(prevMeta.dynamicStyleFields, nextMeta.dynamicStyleFields);
+
+  const updateDueToSourceQuery = !_.isEqual(prevMeta.sourceQuery, nextMeta.sourceQuery);
+
+  const updateDueToIsTimeAware = nextMeta.isTimeAware !== prevMeta.isTimeAware;
+  const updateDueToTime = nextMeta.isTimeAware
```

Comment:
single line. linting enforces the style-guide line-length and will harass if too long.